src/components/layout/UserHomeLayout.tsx

```diff
@@ -0,0 +1,19 @@
+//TODO: Enable eslint rule and fix errors
+/* eslint-disable  @typescript-eslint/no-non-null-assertion */
+import { FunctionComponent } from 'react';
+
+import DefaultLayout from './DefaultLayout';
+import UserHomeMenu from '../UserHomeMenu';
+
+const UserHomeLayout : FunctionComponent = ({ children }) => {
+    return (
+        <>
+            <DefaultLayout>
+                { children }
+            </DefaultLayout>
+            <UserHomeMenu/>
```

Comment:
I feel like `UserHomeMenu` should be contained _inside_ `DefaultLayout` instead of next to it. Compare with `MainOrgLayout`. Does that make sense?